4-projects/business_unit_2/nonprod/example_restricted_shared_vpc_project.tf

```diff
@@ -30,26 +30,10 @@ module "restricted_shared_vpc_project" {
   vpc_service_control_perimeter_name = "accessPolicies/${var.policy_id}/servicePerimeters/${var.perimeter_name}"
 
   # Metadata
-  project_prefix    = "bu2-n-sample"
-  application_name  = "bu2-sample-application2"
+  project_prefix    = "bu2-p-sample-restricted"
```

Comment:
this one should be  `bu2-n-sample-restricted` instead of  `bu2-p-sample-restricted`